discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
+            "tutorials": "yes",
+            "examples": "yes",
+            "eventsAsync": "threaded",
```

Comment:
What I meant is, for example, as I heard, discord.py makes you use coroutines if possible, lest you block too long and heartbeats can't be sent. With discordrb this is a non-issue as event handlers run on separate threads.

In your proposal, what would be the difference between `thread-safe` and `threaded`? As I see it, all libs that use threads in this case *have* to be reasonably thread-safe or it won't work at all.